scripts/lib/getBoundsOfSvgJson.ts

```diff
@@ -82,8 +82,8 @@ export function getBoundsOfSvgJson(svgJson: INode): {
 
   processNode(svgJson)
 
-  const width = Math.max(maxX - minX, 1)
-  const height = Math.max(maxY - minY, 1)
+  const width = Math.abs(maxX - minX)
+  const height = Math.abs(maxY - minY) - 0.35
```

Comment:
Why minus 0.35?

Reply:
Even after the `abs`, the height felt a bit more for most of the symbols. Around 0.35 was the number which made the symbols look decent

Reply:
Should have made a variable with description about this